rotkehlchen/chain/ethereum/modules/eth2/decoder.py

```diff
@@ -94,11 +105,54 @@ def _decode_eth2_deposit_event(self, context: DecoderContext) -> DecodingOutput:
         )
         return DEFAULT_DECODING_OUTPUT
 
+    def _decode_eth2_consolidation_request(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a request to consolidate validators.
+        Handles two types of requests:
+        - Self consolidation - changes the withdrawal credentials to 0x02 (accumulating validator)
+          See https://epf.wiki/?ref=blog.obol.org#/wiki/pectra-faq?id=q-i-have-a-validator-with-0x01-credentials-how-do-i-move-to-0x02
+        - Normal consolidation - consolidates the source validator into the target validator
+        """
+        if len(context.tx_log.data) != 116:  # This an anonymous tx log so can't check topic[0]
+            return DEFAULT_DECODING_OUTPUT
+
+        for event in context.decoded_events:
+            if (
+                event.address == CONSOLIDATION_REQUEST_CONTRACT and
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.asset == A_ETH
+            ):
+                event.event_subtype = HistoryEventSubType.FEE
+                event.counterparty = CPT_ETH2
+                event.notes = f'Spend {event.amount} ETH as validator consolidation fee'
+
+        caller_address = bytes_to_address(b'\x00' * 12 + context.tx_log.data[:20])
+        source_public_key = Eth2PubKey(encode_hex(context.tx_log.data[20:68]))
```

Comment:
I know we had it before but we have `bytes_to_hexstr` that does the same. Maybe we can get rid of one of them but I would use `bytes_to_hexstr` here because it raises deserializationError and the other I suspect raises ValueError